src/core/fetch/index.js

```diff
@@ -174,7 +174,14 @@ export function fetchMixin(proto) {
     if (fallbackLanguages.indexOf(local) === -1) {
       return false
     }
-    const newPath = path.replace(new RegExp(`^/${local}`), '')
+
+    var defaultLanguage = '';
```

Comment:
```suggestion
    var defaultLanguage = defaultFallbackLanguage ? defaultFallbackLanguage : '';
```
and remove the if stmt.

Reply:
In this case you could convert this to a const and not a let as [mentioned](https://github.com/docsifyjs/docsify/pull/916/files#r322351893)